src/howard/howard/templates/howard/certificate.html

```diff
@@ -0,0 +1,69 @@
+{% load i18n %}
+{% load static %}
+
+<html>
+  {% if debug %}
+  <head>
+    <style>
+      {{ css }}
+    </style>
+  </head>
+  {% endif %}
+  <body>
+    <div id="certificate">
+      <header>
+        <div class="logos">
+          <div class="fun">
+            <img src="{{ debug | yesno:",file://" }}{% static "howard/logo-fun.png" %}" />
+          </div>
+          <div class="organization">
+            <img src="{{ debug | yesno:",file://" }}{{ course.organization.logo }}" />
+          </div>
+        </div>
+        <div class="title">
+          <h1>{% translate "Certificate" %}</h1>
+        </div>
+      </header>
+      <article class="main">
+        <section id="representative">
+          <p>
+            {% blocktranslate trimmed with reprensentative=course.organization.representative organization=course.organization.name %}
+              I, {{ reprensentative }},
+              <br>legal representative of the course provider {{ organization }}
+              <br>certifies that:
```

Comment:
I often treat `br` tag as it if was a carriage return. WDYT?

```suggestion
              I, {{ reprensentative }},<br/>
              legal representative of the course provider {{ organization }}<br/>
              certifies that:
```

Reply:
Yes it's better